src/main/webapp/emergancy.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+
+<head>
+	<title>
+		Emergency Button
+	</title>
+	<link rel="stylesheet" href="style_emergancy.css">
+</head>
+
+<body>
+	<!-- Drop down emergancy menu -->
+	<div class="dropdown">
```

Comment:
On user experience/flow, I would expect "emergency" to be a one-click  action that goes to something like a triage screen probably specific to a suspected overdose. Addiction and Resources seem like information for non-emergency scenarios unless resources is really covering who to contact or what to do in case of an emergency.